### ISSUE TITLE
Mapping back 3 sets of reads/sample with minimap2

### ISSUE THREAD

**Issue by kellifeeser** [RasmussenLab/vamb#104]
I used FaQC to qc my raw fastqs before assembling. That program (and perhaps others) outputs properly paired Forward and Reverse fastqs, as well as an unpaired fastq file for each sample. I used the all 3 for each single sample assembly. Since minimap2 only allows for 2 query files, and you mention that "all BAM records for a single read must be consecutive in the BAM file", is it possible to create a bam file for the F and R reads, then a second with the unpaired reads, and finally merge them into a single bam file per sample? Is there a special way to do this so that the downstream steps still run properly? Thank you.

**Comment by simonrasmu** [comment]
Yes, this is possible to map paired and single reads separately and then merge the BAM files. I am not sure how the internal BAM parser in VAMB works on that, however if you use the workflow/jgi counting then it should be no problem at all. 

Additionally, if the single end reads are very few compared to the paired end you could omit for the mapping and binning probably without any loss in bins.

**Comment by TomasaSbaffi** [comment]
Hello,
thanks for the awesome tool! I also would like to use vamb on my assembled data, I use spades --meta to assemble:
-1 non-combined forward reads
-2 non-combined reverse reads
-merged fragments (with flash2)
I wanted to ask if with such input data it is ok to use the workflow, which does not allow to (minimap2) map to 3 read pools but only to forward and reverse (samples2data.txt) or if a more correct approach would be to:
-not use the workflow and
-map to paired and merged reads and then merge the BAM files as the prev user described. 
In my case the merged reads are not few.
thanks a lot
